src/Microsoft.Performance.Toolkit.Engine/EngineCreateInfo.cs

```diff
@@ -80,5 +102,23 @@ public string ExtensionDirectory
         ///     behavior.
         /// </summary>
         public VersionChecker Versioning { get; internal set; }
+
+        /// <summary>
+        ///     Gets or sets the name of the runtime on which the application is built.
+        /// </summary>
+        /// <remarks>
+        ///     Defauls to "Microsoft.Performance.Toolkit.Engine".
+        /// </remarks>
+        public string RuntimeName { get; set; } = "Microsoft.Performance.Toolkit.Engine";
```

Comment:
Do we want to add any validation against null / whitespace here?

Reply:
The engine handles this. I think null/empty should be ok here.

                string runtimeName = !string.IsNullOrWhiteSpace(createInfo.RuntimeName)
                    ? createInfo.RuntimeName
                    : "Microsoft.Performance.Toolkit.Engine";